cterasdk/common/utils.py

```diff
@@ -8,11 +8,25 @@ def union(g1, g2):
     """
     Create a union of two lists, without duplicates.
 
-    :return list: the merged dictionary
+    :return list: the union of the two lists
     """
     return g1 + list(set(g2) - set(g1))
 
 
+def merge(d1, d2):
```

Comment:
Maybe the docstring should indicate that the order matters. That is if the key exists in both dictionaries, the value in the returned dictionary will be from the second one

Reply:
I agree. Fixed. Should I have implemented an *argv function instead of limiting to two parameters? I guess we can optimize it'll become necessary/relevant